README.md

```diff
@@ -1,3 +1,56 @@
-# Health-GPS example configurations
+# Health-GPS example model configurations
 
-Example project configurations and data for Health-GPS
+This repo contains example configurations for the [Health-GPS](../healthgps/) console program, which should cover all of the models currently implented.
+
+## Imperial HPC users
+
+For users of [Imperial College's HPC system](https://www.imperial.ac.uk/computational-methods/hpc/) there is [an example PBS job script](example-jobscript.sh) in this repository.
+
+To get started, follow these instructions after logging into the HPC:
+
+1. Clone the `healthgps` repository:
+
+    ```sh
+    git clone https://github.com/imperialCHEPI/healthgps.git
+    ```
+
+1. Install [CMake] using the [binary distribution on the CMake website] [^1]
+
+1. Load the build tools
+
+    ```sh
+    module load tools/dev
+    module load GCCcore/12.3.0
+    ```
+
+1. Install [vcpkg] following [these instructions] and set the `VCPKG_ROOT` environment variable to point to the `vcpkg` repo in your `.profile`
```

Comment:
Should we provide an example here?

Reply:
You mean of how to set the `VCPKG_ROOT` variable? It is a bit vague atm...